### DISCUSSION TITLE
Invalid version number after Ventura upgrade

### DISCUSSION THREAD

**Discussion by canecaster** [Homebrew/discussions#3832] [Everyday usage] [answered]
First, apologies from a pretty novice Homebrew user.  After upgrading to Ventura on an Intel Mac, I'm getting errors when running gcc (clang: error: invalid version number in '-mmacosx-version-min=13.0').  I've uninstalled and reinstalled Homebrew, and reinstalled gcc.  I didn't see any update of Xcode or command line tools available, though, and wondering if that's my issue.  I did see a discussion here relating to Xcode 14.1 being needed and as yet unreleased, but I'm not enough of a Homebrew user to know if that's what I need to wait for.  Any help would be appreciated.

I've also got an Apple Silicon Mac I need to update to Ventura, but as that is my primary work machine, I didn't want to mess with that one first.

**Comment by p-linnane** [comment, marked as answer]
You will need to download the Xcode 14.1 release candidate or command line tools release candidate. Apple normally releases those the same day as a new OS, but did not do so this year.

**Reply by A5kar** [reply]
Hi @p-linnane, I came across this post for a similar problem but on another package manager (I am using gentoo prefix on Mac M1). Some of my ruby gems are not compiling anymore, and apparently the compiler used in the background is complaining about `<built-in>: error: unknown value '13.0.0' of '-mmacosx-version-min'`. I have been using CLT for Xcode 13.4, but then decided to move to CLT for Xcode 14.1 RC2 and yet still getting the same problem. I wonder if you had to recompile everything on Homebrew (e.g. gcc, ld and friends). Thanks

**Reply by Bo98** [reply]
So `clang: error: invalid version number in '-mmacosx-version-min=13.0'` is a Clang issue that's fixed by updating to Xcode 14.1.

However `<built-in>: error: unknown value '13.0.0' of '-mmacosx-version-min'` is a GCC issue. It should be fixed in GCC 12.2, but has not been backported to earlier GCCs (11, 10, ...) yet. Can you check which version of GCC you are running?

**Reply by A5kar** [reply]
Yep, that was it. Thanks mate!

**Reply by canecaster** [reply]
Thanks folks; this sounds promising.  I don't usually install the betas but I assume the public release of 14.1 will be along in due course.

**Reply by jcvdav** [reply]
Hi all - This problem seems to be resurfacing. The only change to my OS has been Apple's update of [macOS Ventura 13.3.1](https://support.apple.com/en-us/HT213268). I am attempting to install some R packages which require compilation, and getting the error message "`<built-in>: error: unknown value ‘13.3.0’ of ‘-mmacosx-version-min’`". Based on @Bo98's response, this is likely due to GCC. I am running CLT for Xcode 14.3. Do you happen to have any thoughts?
